src/NUnitFramework/nunitlite/TextRunner.cs

```diff
@@ -361,12 +363,52 @@ private int ExploreTests()
                 runSettings[FrameworkPackageSettings.DefaultTestNamePattern] = options.DefaultTestNamePattern;
 
             if (options.TestParameters != null)
-                runSettings[FrameworkPackageSettings.TestParameters] = options.TestParameters;
+            {
+                var dictionary = new Dictionary<string, string>();
```

Comment:
I'll simply wait for the conclusion of [these comments](https://github.com/nunit/nunit-console/pull/155#issuecomment-269808071) and pick up here. =)